http/execute_request.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--method', dest='method', required=True,
+            choices={'GET', 'POST', 'PUT'})
+    parser.add_argument('--url', dest='url', required=True)
+    parser.add_argument('--authorization-header', dest='authorization_header',
+            required=False, default={})
```

Comment:
I'm getting an error `AttributeError: 'str' object has no attribute 'items'` that's related to the default={} for the argument. 
Can we change the default to None, and make the type a string. 
Then, when building the headers dict, include `{'Authorization': args.authorization_header}` if it isn't None.